docs/pages/enroll-resources/application-access/guides/dynamic-registration.mdx

```diff
@@ -117,11 +117,11 @@ $ tctl create app.yaml
 ```
 
 </TabItem>
-<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+<TabItem scope={["cloud"]} label="Teleport Enterprise">
```

Comment:
We need to replace tab items with the "Teleport Cloud" label with "Teleport Enterprise (Cloud)", not "Teleport Enterprise".

Reply:
I will change these:

```
% grep -inr '<TabItem scope={\["cloud"\]} label="Teleport Enterprise">' */| wc -l

      84
```

Reply:
Fixed:

```
% grep -inr '<TabItem scope={\["cloud"\]} label="Teleport Enterprise Cloud">' */| wc -l
      84
```